specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1343,6 +1768,262 @@
+          "readOnly": true,
+          "$ref": "#/definitions/NSPProvisioningState",
+          "description": "The provisioning state of the NSP LinkReference resource."
+        },
+        "remotePerimeterId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Perimeter ARM Id for the remote NSP with which the link is created."
+        },
+        "remotePerimeterGuid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Remote NSP Guid with which the link is created."
+        },
+        "localInboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Local Inbound profile names to which Inbound is allowed. Use ['*'] to allow inbound to all profiles. It's default value is ['*'].",
+          "uniqueItems": true
+        },
+        "localOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Local Outbound profile names from which Outbound is allowed. Use ['*'] to allow outbound from all profiles. It's default value is ['*'].",
+          "uniqueItems": true
+        },
+        "remoteInboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Remote Inbound profile names to which Inbound is allowed. ['*'] value implies inbound is allowed to all profiles at remote perimeter. This property can only be updated from remote perimeter."
+        },
+        "remoteOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Remote Outbound profile names from which Outbound is allowed. ['*'] value implies outbound is allowed from all profiles at remote perimeter. This property can only be updated from remote perimeter."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A message sent by the remote NSP link admin for connection request. In case of Auto-approved flow, it is default to 'Auto Approved'."
+        },
+        "status": {
+          "type": "string",
+          "description": "The NSP linkReference state. It cannot be changed if link is created in auto-approval mode.",
+          "enum": [
+            "Approved",
+            "Pending",
+            "Rejected",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "nspLinkStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Properties of NSP LinkReference."
+    },
+    "NspLinkReferenceReconcile": {
+      "type": "object",
+      "properties": {},
+      "description": "Request object for NSP linkReference reconcile."
     }
   },
```

Comment:
same comment as above , please define a fixed schmea for this

Reply:
Thanks removed it.